### DISCUSSION TITLE
RUntime error

### DISCUSSION THREAD

**Discussion by lefebvre1772** [symfony/symfony#46011] [Q&A]
Hi everyone.
I am nez to symfony and I am running into a problem:
 [RuntimeException]
  Could not delete C:\Users\steph\OneDrive\Dev\dashboard/vendor/composer/tmp-30e5ce1a7623c3d960adeca9f7c903ac:
  This can be due to an antivirus or the Windows Search Indexer locking the file while they are analyzed

Anyone has had that probleme?

**Comment by shmolf** [comment]
I wonder if the issue is related to either the slashes, or the colon at the end.
```
                                     V      V        V                                    V
C:\Users\steph\OneDrive\Dev\dashboard/vendor/composer/tmp-30e5ce1a7623c3d960adeca9f7c903ac:
```
Can you provide more context as to how you acquired that filepath?
